pkg/ocm/ocm.go

```diff
@@ -182,6 +182,24 @@ func (c Client) getClusterResource(clusterID string, resourceKey string) (string
 	return response.Body().Resources()[resourceKey], nil
 }
 
+// GetCloudProvider returns the cloud provider name for a given cluster as a string
+func (c Client) GetCloudProvider(identifier string) (string, error) {
```

Comment:
To avoid confusing it with the `cluster.GetCloudProvider` function. This function wraps the extraction process of `cloudProvider.ID` .

```suggestion
// GetCloudProviderID returns the cloud provider name for a given cluster as a string
func (c Client) GetCloudProviderID(identifier string) (string, error) {
```

Reply:
+1, included the suggested change